tests/components/unifi/test_config_flow.py

```diff
@@ -76,6 +76,7 @@
 WLANS = [
     {"name": "SSID 1"},
     {"name": "SSID 2", "name_combine_enabled": False, "name_combine_suffix": "_IOT"},
+    {"name": "SSID 4", "name_combine_enabled": False, "name_combine_suffix": None},
```

Comment:
Have you looked at your logs if this is the case or if it would be better to do like this

```suggestion
    {"name": "SSID 4", "name_combine_enabled": False},
```

Reply:
When I debugged my case the API always returned None [here](https://github.com/Kane610/aiounifi/blob/master/aiounifi/models/wlan.py#L227) as the attribute is not contained at all in the response of the REST call itself. So from HA point testing against aiounifi I would say my initial proposal is correct, right?

Reply:
In that case it returns none because there is no such key in the data meaning you should omit it as in my suggestion. The most correct thing here is to replicate the response you get from your controller if the key exist and is none or if it's non exististant

Reply:
I checked one more and you are right, thanks for the hin